src/patternfly/demos/DataTable/examples/data-table-compact-demo.hbs

```diff
@@ -0,0 +1,29 @@
+{{#> background-image}}
+{{/background-image}}
+{{#> page page--id="page-layout-data-table-compact"}}
+  {{#> page-header}}
+    {{> data-table-page-header}}
+  {{/page-header}}
+  {{#> page-sidebar}}
+    {{> data-table-page-nav}}
+  {{/page-sidebar}}
+  {{#> page-main}}
+    {{#> page-main-section page-main-section--modifier="pf-m-light"}}
+      {{#> grid grid--modifier="pf-m-gutter"}}
```

Comment:
Why use `grid` here? I don't think you need `grid` in these demos at all. Also you should use `.pf-c-page__main-nav` instead of `.pf-c-page__main-section` to hold the tertiary nav. It applies less top padding so the nav sits closer to the header.

Reply:
BTW the datalist demos have the same problem.

Reply:
thanks for letting me know about that! I'll create an issue to update data list.